logging/s3Helper.go

```diff
@@ -31,22 +32,25 @@ type s3Helper struct {
 	bucket  string
 }
 
+const s3UploadBucket = "algorand-uploads"
```

Comment:
UploadBucket and Region can have defaults but need to be overridable from the environment.

Reply:
OK, updated to allow overriding with defaults with S3_UPLOAD_BUCKET and S3_REGION environment variables.